### PR TITLE
feat(fs-cross-account): Support for FS cross account access

### PR DESCRIPTION
**Pull request by deepaksibm**

### CODE REVIEW

File: ibm/service/vpc/resource_ibm_is_share_accessor_binding_operations.go [hunk 1]
```diff
@@ -0,0 +1,123 @@
+// Copyright IBM Corp. 2021 All Rights Reserved.
+// Licensed under the Mozilla Public License v2.0
```
Comment (ujjwal-ibm) [inline ibm/service/vpc/resource_ibm_is_share_accessor_binding_operations.go:2]: missing test files for this

File: ibm/service/vpc/data_source_ibm_is_share_test.go [hunk 2]
```diff
@@ -72,6 +72,9 @@ func TestAccIbmIsShareDataSourceAllArgs(t *testing.T) {
 					resource.TestCheckResourceAttrSet("data.ibm_is_share.is_share", "zone"),
 					resource.TestCheckResourceAttr("data.ibm_is_share.is_share", "tags.0", "sr1"),
 					resource.TestCheckResourceAttr("data.ibm_is_share.is_share", "tags.1", "sr2"),
+					resource.TestCheckResourceAttrSet("data.ibm_is_share.is_share", "accessor_binding_role"),
```
Comment (ujjwal-ibm) [inline ibm/service/vpc/data_source_ibm_is_share_test.go:75]: is this working ?

File: ibm/service/vpc/data_source_ibm_is_shares_test.go [hunk 3]
```diff
@@ -52,6 +52,9 @@ func TestAccIbmIsSharesDataSourceAllArgs(t *testing.T) {
 					resource.TestCheckResourceAttrSet("data.ibm_is_shares.is_shares", "shares.0.name"),
 					resource.TestCheckResourceAttrSet("data.ibm_is_shares.is_shares", "shares.0.resource_type"),
 					resource.TestCheckResourceAttrSet("data.ibm_is_shares.is_shares", "shares.0.size"),
+					resource.TestCheckResourceAttrSet("data.ibm_is_shares.is_shares", "shares.0.accessor_binding_role"),
```
Comment (ujjwal-ibm) [inline ibm/service/vpc/data_source_ibm_is_shares_test.go:55]: is this working ?

**Review by ujjwal-ibm — CHANGES_REQUESTED** [review]

**Comment by deepaksibm** [comment]
SDK released in public repo

**Comment by deepaksibm** [comment]
$ tfpt TestAccIbmIsShareDataSourceAllArgs
=== RUN   TestAccIbmIsShareDataSourceAllArgs
--- PASS: TestAccIbmIsShareDataSourceAllArgs (129.24s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     131.318s


$ tfpt TestAccIbmIsSharesDataSourceAllArgs
=== RUN   TestAccIbmIsSharesDataSourceAllArgs
--- PASS: TestAccIbmIsSharesDataSourceAllArgs (89.76s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     91.817s

**Comment by deepaksibm** [comment]
```

$ tfpt TestAccIbmIsShareOriginShare
=== RUN   TestAccIbmIsShareOriginShare
--- PASS: TestAccIbmIsShareOriginShare (140.51s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     142.736s
```

**Comment by deepaksibm** [comment]
```
$ tfpt TestAccIBMIsShareAccessorBindingsDataSourceBasic
=== RUN   TestAccIBMIsShareAccessorBindingsDataSourceBasic
--- PASS: TestAccIBMIsShareAccessorBindingsDataSourceBasic (99.14s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     101.190s

$ tfpt TestAccIBMIsShareAccessorBindingDataSourceBasic
=== RUN   TestAccIBMIsShareAccessorBindingDataSourceBasic
--- PASS: TestAccIBMIsShareAccessorBindingDataSourceBasic (100.81s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     102.786s

$ tfpt TestAccIbmIsShareDeleteAccessorBinding
=== RUN   TestAccIbmIsShareDeleteAccessorBinding
--- PASS: TestAccIbmIsShareDeleteAccessorBinding (149.14s)
PASS
ok      github.com/IBM-Cloud/terraform-provider-ibm/ibm/service/vpc     151.143s
```

File: ibm/service/vpc/resource_ibm_is_share_accessor_binding_operations.go (hunk 1 above)
Comment (deepaksibm) [reply ibm/service/vpc/resource_ibm_is_share_accessor_binding_operations.go]: modified resource to ibm_is_share_delete_accessor_binding
added test file

**Comment by deepaksibm** [comment]
upstream PR merged

**Pull request closed without merging**